src/util/validate.py

```diff
@@ -15,6 +15,20 @@
 }
 
 
+def validate_assessment_id(assessment_id):
+    """Validate that the provided assessment_id is matching the valid assessment_id format. Example: RV1234.
+
+    Args:
+        assessment_id (string): Assessment identifier to validate.
+
+    Returns:
+        match: the result of a regular expression match.
```

Comment:
Use consistent capitalization:
```suggestion
        match: The result of a regular expression match.
```

Reply:
See c6d29c5 for revisions